tests/testsuite/artifact_dep.rs

```diff
@@ -0,0 +1,2024 @@
+use cargo_test_support::compare::match_exact;
```

Comment:
Can this include a doc comment at the top explaining what this is for (see other modules for examples).

Reply:
A valid point, I tried to be descriptive while not going over board with details here: 93ddd285156cae829f32887de6e5e07746529b1e .